src/qcodes/dataset/data_set.py

```diff
@@ -1052,7 +1059,11 @@ def to_xarray_dataset(
                 if None
             end: end value of selection range (by results count); ignored if
                 None
-
+            use_multi_index: Should the data be exported using a multi index.
+                If the data is not on a grid this will create a xarray dataset
+                without missing datapoints. If set to "auto" multi index will be
```

Comment:
"without missing datapoints" - could this be said differently such that the word "missing" does not trigger thinking about data points that were missed during a measurement or alike? perhaps "... only with datapoints that are existing in the dataset, that is without trying to fit the data onto a grid thus creating a very sparsely-popuated grid" although it's lengthy. Is there any vocabulary we can take from xarray?))